pkg/config/app.go

```diff
@@ -94,28 +98,42 @@ func GetInstance() *AppConfig {
 }
 
 func (c *AppConfig) setDefaultConfig() {
+
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatal("Error loading .env file")
```

Comment:
This means .env must be present, which - is okay - but different than before, so that the current docker setup won't work. I can edit the docker setup, or - we can go back to allowing the .env file to be not present, and relying on cli args, env vars, a config file, or built in defaults to set up required config vars